src/bFS/Node.java

```diff
@@ -0,0 +1,26 @@
+package bFS;
+
+import java.util.ArrayList;
+
+public class Node {
+	
+	private ArrayList<Node> children;
+	private int value;
+	
+	public Node(int v) {
+		value = v;
+		children = new ArrayList<Node>();
+	}
+	
+	public void addChild(Node n) {
+		children.add(n);
+	}
+	
+	public ArrayList<Node> getChildren() {
+		return children;
+	}
+	
+	public int getValue() {
+		return value;
+	}
```

Comment:
good AP habit that would get you points on the AP. If you forget this, you are going to get points off on the AP. However, I also want to remind you that this is not necessary, so if you are at a competition and the faster you get the code working, the more points you have, then don't do this. You can simply make the variables `public` and directly modify them.